components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/internal/SyncedTabsController.kt

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.syncedtabs.internal
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import mozilla.components.feature.syncedtabs.SyncedTabsStorage
+import mozilla.components.feature.syncedtabs.view.SyncedTabsView
+import mozilla.components.feature.syncedtabs.view.SyncedTabsView.ErrorType
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import mozilla.components.service.fxa.manager.ext.withConstellation
+import kotlin.coroutines.CoroutineContext
+
+internal class SyncedTabsController(
+    private val storage: SyncedTabsStorage,
+    private val accountManager: FxaAccountManager,
+    private val view: SyncedTabsView,
+    private val coroutineContext: CoroutineContext
+) {
+
+    fun syncTabs() {
```

Comment:
`syncTabs` implies that we're syncing the tabs, whereas we display them if they are available. We should consider another name (e.g. `displayTabs`).

Reply:
This does the 'syncing' and `SyncedTabsView.displaySyncedTabs` does the displaying.